CorsixTH/Lua/dialogs/resizables/main_menu.lua

```diff
@@ -96,6 +96,12 @@ function UIMainMenu:draw(canvas, x, y)
     ly = ly - 15
   end
   self.label_font:draw(canvas, _S.main_menu.version .. self.version_number, x + 5, ly, 190, 0, "right")
+  TheApp:idle()
```

Comment:
This could potentially be inside `App:onTick()`, but then that function call happens every tick the game is open.